### PR TITLE
API for importing multiple Excel Sheets in an Excel Book.

### PR DESCRIPTION
**Pull request by hidekoji**
# Description

Add APIs to import multiple Excel Worksheets in an Excel Book as a single data frame.

# Checklist
Make sure you have performed the following items before submitting this pull request.
If not, please describe the reason.  

- [ ] Add test cases for this fix/enhancement
- [ ] Pass devtools::check()
- [ ] Pass devtools::test()
- [ ] Test installing from github
- [x] Tested with Exploratory

### CODE REVIEW

File: R/system.R [hunk 1]
```diff
@@ -2976,7 +2976,21 @@ download_data_file <- function(url, type){
   }
 }
 
-#'API that search and imports multiple same structure Excel files and merge it to a single data frame
+#'API that searches and imports multiple same structure Excel Sheets in the Excel Book and merge them to a single data frame
+#'@export
+searchAndReadExcelFileMultiSheets <- function(file, forPreview = FALSE, pattern = "", col_names = TRUE, col_types = NULL, na = "", skip = 0, trim_ws = TRUE, n_max = Inf, use_readxl = NULL, detectDates = FALSE, skipEmptyRows = FALSE, skipEmptyCols = FALSE, check.names = FALSE, tzone = NULL, convertDataTypeToChar = TRUE, ...) {
+  # search condition is case insensitive. (ref: https://www.regular-expressions.info/modifiers.html, https://stackoverflow.com/questions/5671719/case-insensitive-search-of-a-list-in-r)
+  sheets <- readxl::excel_sheets(file)
+  sheets <- sheets <- sheets[stringr::str_detect(sheets, pattern)]
```
Comment (hideaki) [inline R/system.R:2984]: Fix the assignment.

File: R/system.R [hunk 2]
```diff
@@ -2993,6 +3007,35 @@ searchAndReadExcelFiles <- function(folder, forPreview = FALSE, pattern = "", sh
 
 }
 
+#'API that imports multiple same structure Excel sheets in the Excel Book and merge them to a single data frame
+#'@export
+read_excel_file_multi_sheets <- function(file, forPrevew = FALSE, sheets = c(1), col_names = TRUE, col_types = NULL, na = "", skip = 0, trim_ws = TRUE, n_max = Inf, use_readxl = NULL, detectDates = FALSE, skipEmptyRows = FALSE, skipEmptyCols = FALSE, check.names = FALSE, tzone = NULL, convertDataTypeToChar = TRUE, ...) {
```
Comment (hideaki) [inline R/system.R:3012]: Add comment on the default of sheets argument.

File: R/system.R [hunk 3]
```diff
@@ -2993,6 +3007,35 @@ searchAndReadExcelFiles <- function(folder, forPreview = FALSE, pattern = "", sh
...
+                       skip = skip,
+                       trim_ws = trim_ws,
+                       n_max = n_max,
+                       use_readxl = use_readxl,
+                       detectDates = detecDates,
+                       skipEmptyRows = skipEmptyRows,
+                       skipEmptyCols = skipEmptyCols,
+                       check.names = check.names,
+                       tzone = tzone, convertDataTypeToChar = convertDataTypeToChar),
+                       .id = "exp.file.id") %>% mutate(exp.file.id = basename(exp.file.id)) # extract file name from full path with basename.
```
Comment (hideaki) [inline R/system.R:3030]: Rename the column name exp.file.id appropriately.
basename() is not necessary.

File: R/system.R (hunk 2 above)
Comment (hidekoji) [reply R/system.R:3014]: Added a comment for the "sheets" argument default value.

File: R/system.R (hunk 3 above)
Comment (hidekoji) [reply R/system.R]: I renamed the "exp.file.id" to "exp.sheet.id". 
I also removed the basename() mutate command.

**Pull request merged**